65.md

```diff
@@ -4,73 +4,58 @@ NIP-65
+When broadcasting an event, Clients SHOULD:
 
-### Motivation
+- Broadcast the event to the WRITE relays of the author
+- Broadcast the event all READ relays of each tagged user. 
 
-There is a common nostr use case where users wish to follow the content produced by other users. This is evidenced by the implicit meaning of the Contact List in [NIP-02](02.md)
+## Motivation
 
-Because users don't often share the same sets of relays, ad-hoc solutions have arisen to get that content, but these solutions negatively impact scalability and decentralization:
+The old model of using a fixed relay list per user centralizes in large relay operators: 
 
-  - Most people are sending their posts to the same most popular relays in order to be more widely seen
-  - Many people are pulling from a large number of relays (including many duplicate events) in order to get more data
+  - Most users submit their posts to the same highly popular relays, aiming to achieve greater visibility among a broader audience.
+  - Many users are pulling events from a large number of relays in order to get more data at the expense of duplication
   - Events are being copied between relays, oftentimes to many different relays
+  
+This NIP allows Clients to connect directly with the most up-to-date relay set from each individual user, eliminating the need of broadcasting events to popular relays. 
 
-### Purposes
-
-The purpose of this NIP is to help clients find the events of the people they follow, to help tagged events get to the people tagged, and to help nostr scale better.
+## Final Considerations
 
-### Suggestions
+1. Clients SHOULD guide users to keep `kind:10002` lists small (2-4 relays). 
 
-It is suggested that people spread their kind `10002` events to many relays, but write their normal feed-related events to a much smaller number of relays (between 2 to 6 relays). It is suggested that clients offer a way for users to spread their kind `10002` events to many more relays than they normally post to.
+2. Clients SHOULD spread an author's `kind:10002` events to as many relays as viable. 
 
-Authors may post events outside of the feed that they wish their followers to follow by posting them to relays outside of those listed in their "Relay List Metadata".  For example, an author may want to reply to someone without all of their followers watching.
+3. `kind:10002` events MUST only be used to advertise the user's preferred relays to others. The Client's configuration SHOULD stay in other lists where other factors can be considered, such as specific relays for a given UI feature of the Client.
 
-It is suggested that relays allow any user to write their own kind `10002` event (optionally with AUTH to verify it is their own) even if they are not otherwise subscribed to the relay because
+4. DMs SHOULD only be broadcasted to the author's WRITE relays and to the receiver's READ relays to keep maximum privacy. 
```

Comment:
I think we need some other special rule for DMs. I think ideally DMs would only go to a subset of the relays defined by the receiver, not the sender, so that would be just to _the receiver's READ relays_.

However, even that is not enough, because I may be perfectly willing to read public notes on some relays, but not DMs. For DMs specifically I may want to use one specific relay that I trust more with my metadata, for example.

Maybe add a different marker `"dm-read"` and fallback to the `"read"`?

Reply:
@mikedilger was adding DMs on the other PR. Should I add them here as well? 

Maybe DM is the wrong word. I can imagine a special relay that third parties can post to it but only the author can download from it.

Reply:
This is a bit trickier because you have to read back your own DMs from somewhere to see the full conversation.

Reply:
Your own DMs from WRITE + replies from PROTECTED_READ_RELAYS

Reply:
Maybe protected DM relays are the subject of a new PR when we better understand them.

Reply:
Agreed, let's punt on the special DM stuff for now, take one step at a time.

Reply:
Agreed.